root/release/ReleaseHeader.js

```diff
@@ -41,6 +41,7 @@ const ReleaseHeader = ({release, page}: Props) => {
             <EntityLink
               content={rgLink}
               entity={release.releaseGroup}
+              nameVariation
```

Comment:
Nitpick of course, but it seems weird to call it a name variation here - we're just overriding the name bit completely. Maybe we should call the variable "hasCustomContent" or something else? (but it might not matter at all)

Reply:
Hmm. `hasCustomContent` could be misleading too, since that's already specified by having a `content` prop, whereas this is more about if the content prop has text that differs from the entity name. `textContentDiffersFromEntityName` seems accurate, but might be too wordy. :)

Reply:
Another way to look at it is that `nameVariation` just controls whether the content is wrapped in `<span className="name-variation" />` with hover text added.

Reply:
Fiiine :)